### ISSUE TITLE
New Octane Rule: `no-computed-properties-in-native-classes`

### ISSUE THREAD

**Issue by patocallaghan** [ember-cli/eslint-plugin-ember#544]
Pulled from https://github.com/ember-cli/eslint-plugin-ember/issues/533 which proposes a new set of rules for Octane related best practices

Disallow usage of Computed Properties in native JS classes. `@tracked` and getters should be used instead.

```js
// ### Good
import { tracked } from '@glimmer/tracking';
export default class MyComponent extend Component {
  ...
}

import { computed } from '@ember/object';
import { and, or, alias } from '@ember/object/computed';
export default class Component.extends({
  ...
});

// ### Bad
import { computed } from '@ember/object';
import { and, or, alias } from '@ember/object/computed';
export default class MyComponent extend Component {
  ...
}
```

If anyone would like to implement this please comment that you'll take it and work away.

**Comment by patocallaghan** [comment]
@rwjblue this is one I was proposing. The idea was just look at the imports and throw an error if it's a native class. Thoughts?

**Comment by scalvert** [comment]
This makes sense to me. 👍

**Comment by patocallaghan** [comment]
I'll take this one ✋

**Comment by patocallaghan** [comment]
I've put together a PR for this rule in https://github.com/ember-cli/eslint-plugin-ember/pull/546
